alm/src/hooks/useMessageConfirmations.ts

```diff
@@ -192,8 +194,10 @@ export const useMessageConfirmations = ({
         })
       }
 
-      homeBlockNumberProvider.start(home.web3)
-      const targetBlock = collectedSignaturesEvent.blockNumber + blockConfirmations
+      homeBlockNumberProvider.start(home.web3);
+      (async () => {
```

Comment:
if we're using subgraph, we do not have collectedSignaturesEvent.blockNumber because subgraph data does not include block number. so basically, we're getting the block number of the event from the tx reciept

Reply:
@masterbluecompact got it. We can add a blocknumber to the events, would that help?

Reply:
yeah that would help, but it is okay the way it is right now, but could be better with block number in the subgraph data

Reply:
then i will add the blocknumber to the events